src/test/unit/web_interface/test_app_ajax_routes.py

```diff
@@ -37,6 +38,12 @@ def test_ajax_get_system_stats_error(self):
         assert result['backend_cpu_percentage'] == 'n/a'
         assert result['number_of_running_analyses'] == 'n/a'
 
+    def test_ajax_system_health(self):
+        self.mocked_interface.get_stats_list = lambda *_: [{'foo': 'bar'}]
```

Comment:
Why the different patching compared to the test just above? Overwriting without setting it back afterwards is bad style.